portfolio/src/main/webapp/script.js

```diff
@@ -115,22 +117,45 @@ document.getElementById('finish-button').addEventListener('click', () => {
   const scoreContainer = document.getElementById('score-container');
 
   if (score > questionList.length / 2) {
-    scoreContainer.innerHTML =
+    scoreContainer.textContent =
         `Your score is ${score}! Here is a spotify link for you to enjoy:
             ${SPOTIFY_LINK}`;
 
   } else {
-    scoreContainer.innerHTML = 'Your score is ' + score;
+    scoreContainer.textContent = 'Your score is ' + score;
   }
 });
 
 
-/** Adds a random greeting to the page. */
-function addRandomMovie() {
-  // Pick a random greeting.
-  const movie = MOVIES[Math.floor(Math.random() * MOVIES.length)];
-
-  // Add it to the page.
+/** Adds a random movie list to the page. */
+function addRandomMovies() {
+  const API_KEY= '';
+  const FETCH_URL =
+      `https://api.themoviedb.org/3/movie/popular?api_key=${API_KEY}&language=en-US&page=1`
+  
   const movieContainer = document.getElementById('movie-container');
-  movieContainer.innerText = movie;
-}
\ No newline at end of file
+
+  movieContainer.textContent = 'Loading...';
+
+  const moviePromise = fetch(FETCH_URL);
+
+  moviePromise
+      .then((response) => {
+        return response.json();
+      })
+      .then((data) => {
+        movieContainer.innerHTML = parseJson(data);
```

Comment:
Especially since this is taking a response from an external service, try to avoid using innerHTML.  Instead of writing the tags using strings, use document.createElement('tagName'); and element.appendChild(childElement).

The danger here is that they could easily change their response to something like 
`<div onLoad="attackComputer()">Movie Name</div>`
then when you add it to your page, everything will look normal, but your site will start attacking users

Reply:
Okay, I fixed that.